deployment/deploy_storybook_gitlab.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+PAGES_BRANCH="gh-pages"
+
+echo "open ssl connection"
+eval `ssh-agent -s`
+ssh-add <(echo "$GITHUB_DEPLOY_KEY" | base64 -d)
+mkdir -p ~/.ssh
+ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
```

Comment:
I'd check in the key retrieved via ssh-keyscan.

Reply:
could you elaborate a little more please.